erdpy/cli_contracts.py

```diff
@@ -359,3 +370,28 @@ def _send_or_simulate(tx: Transaction, contract: SmartContract, args: Any):
     output_builder = cli_shared.send_or_simulate(tx, args, dump_output=False)
     output_builder.set_contract_address(contract.address)
     utils.dump_out_json(output_builder.build(), outfile=args.outfile)
+
+
+def reproduce_build(args: Any):
+    project_paths = get_project_paths(args)
+
+    if len(project_paths) == 1:
+        project_path = project_paths[0].expanduser().resolve()
+    else:
+        raise NotImplementedError()
```

Comment:
I think, in this case, a simplification would be to not use `_add_recursive_arg`, nor `get_project_paths()`, and just use `args.project_path`, instead.

Can also be kept as it is, for the moment (`_add_recursive_arg` + `get_project_paths`).

Reply:
Modified to only use `args.project`